assets/js/atomic/blocks/product/summary/index.js

```diff
@@ -14,7 +14,7 @@ import sharedConfig from '../shared-config';
 const blockConfig = {
 	title: __( 'Product Summary', 'woo-gutenberg-products-block' ),
 	description: __(
-		'Display the short description of a product.',
+		'Display a short description about a product.',
```

Comment:
I like this improvement, however this changes a i18n string that has been released (breaking any existing translations). Is that okay? Maybe check to see how well this is translated?

Reply:
Hmm if the string is an improvement I don't see why we should limit ourselves to existing translations. Having to re-translate something isn't a breaking change...

Reply:
Technical/functionality breaking change? No.  International sites/stores relying on correct translations for their customers and not knowing about changes in strings?  Yes.

Reply:
Right but this isn't much different to releasing new functionality, which isn't immediately translated. I don't think we should be locking ourselves into translations indefinitely, especially for editor translations only store owners see.

Reply:
> Right but this isn't much different to releasing new functionality, which isn't immediately translated. 

Hmm, I think there is a difference.  A new feature isn't implemented on most sites yet so I think there's a bit more leeway, whereas with existing features there's already some time for translations to populate and it's more jarring if something that _was_ translated all of a sudden isn't.

> I don't think we should be locking ourselves into translations indefinitely,

I agree, which is why my original feedback was posed as a question. I definitely don't think we should be locked into translations indefinitely but also think we should be careful about when we _do_ break translations that it's definitely needed. We have three languages (as of now) on the Woo Blocks repo that have full string coverage (btw, that's simply amazing!)

>  especially for editor translations only store owners see.

This is a really good point, I think we do have a bit more leeway here because of that.

To be clear, I don't intend for this question to be a blocker.  I'm mostly just wanting to make sure we make changes in translation strings carefully. So with that said, I think your answer to the question "Is that okay?" is yes for these case, and that's good enough for me as I too like improved clarity and I think we're still fairly early enough in the exposure for these strings where the impact should be minimal.